pages/events.js

```diff
@@ -11,26 +11,37 @@ export default class Events extends Component {
         <Helmet
           title={config.siteTitle}
         />
-        <div className="events__title">
+        <div className="page__title">
           Events
         </div>
 
-        <div className="events__content">
-          <div className="content__section">
-            <div className="events__col--1">Upcoming Events</div>
-            <div className="events__col--2">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aliquid voluptate soluta dolorum dignissimos in nostrum eum recusandae similique, at facilis ad quaerat rerum magnam commodi obcaecati, voluptas magni culpa quidem.
+        <div className="page__content">
+            <div className="col col--1">Upcoming Events</div>
+            <div className="col col--2">
+              <div className="col--2__title">November 13th - Submission deadline for the International Relations Review of Berkeley.</div>
+              <div className="col--2__section">Click here (link to IRC Review page) for more information about the submission process.
```

Comment:
I think they want a link to the IRC review page here.

```html
Click <a href={ prefixLink(/review/) }>here</a> for more information...
```

prefixLink is used whenever we want to link to something in Gatsby.  I'll explain more in detail in person.  We want to link to ```/review/``` because it is the relative path to the review page from the current page we are at right now, which is ```events.js```